search-sentiments-map/routes/search.js

```diff
@@ -186,16 +187,16 @@ async function updateSearchResults() {
   let countries = countriesJson.map(country => country.id);
 
   const trends = await search.retrieveGlobalTrends();
-  for (let i = 0; i < trends.length; i++) {
+  for (let i = 0; i < trends.length; i++) { 
```

Comment:
```suggestion
  for (let i = 0; i < trends.length; i++) {
```

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace